public/src/app/applications/app-public-notices/public-notices-panel.component.html

```diff
@@ -121,7 +121,7 @@ <h3 class="title">No public notices found.</h3>
 
       <div class="row">
         <div class="col-sm-4 bold">Validity Period: </div>
-        <div class="col-sm-8">{{pn.project.commentingOpenDate | date: 'longDate'}} to {{pn.project.validityEndDate | date: 'longDate'}}</div>
+        <div class="col-sm-8">{{(pn.project.bctsMgrName? pn.project.commentingClosedDate : pn.project.commentingOpenDate) | date: 'longDate'}} to {{pn.project.validityEndDate | date: 'longDate'}}</div>
```

Comment:
I'd add a validatyStartDate instead of having this logic in the view.

Reply:
Yes, better not in view for long logic. Thanks Basil!